WordPress/src/main/res/layout/activity_log_list_loading_item.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_height="wrap_content"
+    android:layout_width="match_parent"
+    android:minHeight="?attr/listPreferredItemHeightLarge" >
+
+    <ProgressBar
+        android:id="@+id/progress"
+        android:layout_centerInParent="true"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_extra_large"
+        android:layout_width="wrap_content"
+        android:visibility="invisible"
```

Comment:
this could be removed

Reply:
This was resolved with [this comment] (https://github.com/wordpress-mobile/WordPress-Android/pull/8372#issuecomment-424982372)